src/Maker/MakeStimulusController.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\WebpackEncoreBundle\WebpackEncoreBundle;
+
+/**
+ * @author Abdelilah Jabri <jbrabdelilah@gmail.com>
+ *
+ * @internal
+ */
+final class MakeStimulusController extends AbstractMaker
+{
+    public static function getCommandName(): string
+    {
+        return 'make:stimulus-controller';
+    }
+
+    public static function getCommandDescription(): string
+    {
+        return 'Creates a new Stimulus controller';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConfig)
+    {
+        $command
+            ->addArgument('name', InputArgument::REQUIRED, 'The name of the Stimulus controller (e.g. <fg=yellow>hello</>)')
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeStimulusController.txt'));
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        $command->addArgument('extension', InputArgument::OPTIONAL);
+        $command->addArgument('targets', InputArgument::IS_ARRAY, '', []);
+
+        $chosenExtension = $io->choice(
+            'Language (<fg=yellow>JavaScript</> or <fg=yellow>TypeScript</>)',
+            [
+                'js' => 'JavaScript',
+                'ts' => 'TypeScript',
+            ]
+        );
+
+        $input->setArgument('extension', $chosenExtension);
+
+        $includeTargets = $io->confirm('Do you want to include targets?');
+        if ($includeTargets) {
+            $targets = [];
+            $isFirstTarget = true;
+            while (true) {
+                $newTarget = $this->askForNextTarget($io, $targets, $isFirstTarget);
+                $isFirstTarget = false;
+
+                if (null === $newTarget) {
+                    break;
+                }
+
+                $targets[] = $newTarget;
+            }
+
+            $input->setArgument('targets', $targets);
+        }
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    {
+        $controllerName = $input->getArgument('name');
```

Comment:
$controllerName perhaps need to be rewriten according to the user input 

if the user write ProductSearch or productSearch it should be product_search at the end i think

Reply:
@94noni I added Str::asSnakeCase